roles/ceph-nfs/tasks/pre_requisite_container.yml

```diff
@@ -26,16 +26,3 @@
         - item.item.copy_key | bool
   when: groups.get(mon_group_name, []) | length > 0
 
-- name: dbus related tasks
-  block:
-    - name: create dbus service file
-      copy:
-        src: "org.ganesha.nfsd.conf"
-        dest: /etc/dbus-1/system.d/org.ganesha.nfsd.conf
```

Comment:
I confirmed that this file exists in the nfs-ganesha package 2.7 (luminous), 2.8 (nautilus) and 3.0 (octopus)

Reply:
ok maggle, thanks!